### PR TITLE
docs: Common actions refresh

### PR DESCRIPTION
**Pull request by connor-leech**
Changes based off of: https://github.com/carbon-design-system/carbon-website/issues/815

### CODE REVIEW

File: src/pages/patterns/common-actions/index.mdx [hunk 1]
```diff
@@ -37,21 +67,21 @@ Use a secondary button or a link for cancel actions.
 </Column>
 </Row>
 
-### Clear
+## Clear
 
-Clear removes data from a field or removes selections. Clear can also delete the contents of a document, such as a log. Typically, the default selection or value is reset for controls that have a default selection or value, such as radio buttons.
+Clear removes data from a field or removes selections. Clear can also delete the contents of a document, such as a log. For controls that have a default selection or value, such as radio buttons, the default selection or value is reset.
 
 Use the `x` icon on the right side of a field, item, or value.
```
Comment (connor-leech) [inline src/pages/patterns/common-actions/index.mdx]:
```suggestion
Use the `close` icon on the right side of a field, item, or value.
```

File: src/pages/patterns/common-actions/index.mdx [hunk 2]
```diff
@@ -5,261 +5,252 @@ description: Common actions are frequently used actions that appear across diffe
...
 </Row>
 
-### Edit
+### Content guidelines
 
-Edit allows data or values to be changed. Edit commonly triggers a state change to the targeted object or input item.
+Be brief, honest, supportive, and human. Explain what happened and what can the user do to resolve the error.
 
-Offer edit as an option in a menu, or as a button or `edit` icon.
+For full-page and large modals, keep error messages no longer than three paragraph lines. For form errors, keep error messages no longer than two lines.
```
Comment (jillianhowarth) [inline src/pages/patterns/common-actions/index.mdx]: I think the content guidelines are meant to go under errors

File: src/pages/patterns/common-actions/index.mdx [hunk 3]
```diff
@@ -5,261 +5,252 @@ description: Common actions are frequently used actions that appear across diffe
...
-### Refresh
+### Hierarchy and placement
 
-This action reloads the view of an object, list, or data set when the displayed view has become unsynchronized with the source.
+Represent remove as a button or `subtract` icon or glyph. The remove action is rarely the primary action on the page and should not be overly emphasized.
 
-Use the `refresh` icon or a button.
+### Considerations
+
+- What are the implications of the add action for the user? Are there financial, access, or legal considerations?
```
Comment (jillianhowarth) [inline src/pages/patterns/common-actions/index.mdx]:
```suggestion
- What are the implications of the remove action for the user? Are there financial, access, or legal considerations?
```

File: src/pages/patterns/common-actions/index.mdx (hunk 2 above)
Comment (connor-leech) [reply src/pages/patterns/common-actions/index.mdx]: Good catch, thank you!

**Pull request merged**